tests/UI/campaigns/functional/FO/07_search/01_consultAutocompleteList.js

```diff
@@ -49,63 +35,70 @@ describe('FO - Search Page : Search product and consult autocomplete list', asyn
     await helper.closeBrowserContext(browserContext);
   });
 
-  it('should login in BO', async function () {
-    await loginCommon.loginBO(this, page);
-  });
-
-  it('should go to \'Shop parameters > Search\' page', async function () {
-    await testContext.addContextItem(this, 'testIdentifier', 'goToSearchPageToDisable', baseContext);
-
-    await dashboardPage.goToSubMenu(page, dashboardPage.shopParametersParentLink, dashboardPage.searchLink);
-
-    const pageTitle = await searchPage.getPageTitle(page);
-    await expect(pageTitle).to.contains(searchPage.pageTitle);
-  });
+  it('should go to FO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToFO', baseContext);
 
-  it('should disable fuzzy search', async function () {
-    await testContext.addContextItem(this, 'testIdentifier', 'DisableFuzzySearch', baseContext);
+    await homePage.goToFo(page);
 
-    const result = await searchPage.setFuzzySearch(page, false);
-    await expect(result).to.contains(searchPage.settingsUpdateMessage);
+    const isHomePage = await homePage.isHomePage(page);
+    await expect(isHomePage).to.be.true;
   });
 
-  it('should go to FO and search product to check the autocomplete list', async function () {
+  it('should check the autocomplete list', async function () {
     await testContext.addContextItem(this, 'testIdentifier', 'checkAutocompleteList', baseContext);
 
-    page = await searchPage.viewMyShop(page);
+    const searchValue = 'test';
+    const numSearchResults = 7;
 
-    const isHomePage = await homePage.isHomePage(page);
-    await expect(isHomePage).to.be.true;
+    const numResults = await homePage.countAutocompleteSearchResult(page, searchValue);
+    await expect(numResults).equal(numSearchResults);
 
-    const result = await homePage.getAutocompleteSearchResult(page, searchValue);
+    const inputValue = await homePage.getInputValue(page, homePage.searchInput);
+    await expect(inputValue).equal(searchValue);
 
-    for (let i = 0; i < searchResults.length; i++) {
-      await expect(result, `Search result should contain ${searchResults[i]}`).to.contain(searchResults[i]);
-    }
+    await homePage.closeAutocompleteSearch(page);
```

Comment:
```suggestion
```

Maybe moving close action to the next step

Reply:
Fixed